docs/getting-started/my-page/README.md

```diff
@@ -56,24 +56,30 @@ You can change the position of a widget on the dashboard with drag and drop.
 
 Click the dots next to the title and drag it to the new place.
 
-![add-widget-my-page](add-widget-my-page2.gif)
+<video src="/Users/jankeppler/Desktop/Bildschirmaufnahme 2022-02-25 um 13.13.10.mov"></video>
```

Comment:
This video is only on your computer and it will not work in many browsers, since mov is not supported by many. Please create animated gifs.